socorro/external/es/crashstorage.py

```diff
@@ -119,6 +120,29 @@ def save_raw_and_processed(self, raw_crash, dumps, processed_crash,
             'raw_crash': raw_crash
         }
 
+        # NOTE(willkg): this is a hard-coded keyname to match what the statsdbenchmarkingwrapper
+        # produces for processor crashstorage classes. This is only used in that context, so we're
+        # hard-coding this now rather than figuring out a better way to carry that name through.
+        try:
+            self.config.metrics.histogram(
+                'processor.es.raw_crash_size',
+                len(json.dumps(raw_crash, cls=JsonDTEncoder))
```

Comment:
I don't know offhand what the performance consequences of serializing these again are. I'm hoping it's not too much of a change compared to the time it normally would take. If it ends up taking more time than we'd like, we can adjust this code to sample metrics gathering and only capture 1/100 or something like that.